README.md

```diff
@@ -1,6 +1,6 @@
-# benchmarks
+# examples
```

Comment:
I know it wasn't before, but this should probably be capitalized?

Reply:
Sure.